### DISCUSSION TITLE
https://numbat.dev/ behavior on mobile

### DISCUSSION THREAD

**Discussion by klkuhlm** [sharkdp/numbat#410] [Ideas]
I like using numbat, and have tried to use it on my phone (iphone in either safari or chrome) several times, but it is difficult. 

When you start to type at the prompt (">>>"),  it immediately scrolls away off the top of the page. It seems the browser is centering the box, rather than going to the top where the prompt is.  After you enter a couple things, the prompt becomes visible. It is disorienting, though.

I am not sure if mobile behavior is important or possible to fix, but I thought I would bring it up somewhere.

**Comment by sharkdp** [comment]
Please see https://github.com/sharkdp/numbat/issues/218

**Reply by klkuhlm** [reply]
Sorry I missed this. Looking forward to the mobile version
